src/main/java/io/suhan/racingcar/Game.java

```diff
@@ -1,71 +1,38 @@
 package io.suhan.racingcar;
 
-import java.util.ArrayList;
 import java.util.List;
 
 public class Game {
-    private final List<Car> cars;
-    private GameState state;
-    int rounds;
+    private final CarRegistry carRegistry;
+    private int rounds;
 
     public Game() {
-        this.cars = new ArrayList<>();
-        this.state = GameState.IDLE;
+        this.carRegistry = new CarRegistry();
         this.rounds = 0;
     }
 
-    public void registerCar(Car car) {
-        this.cars.add(car);
-    }
-
     public void start() {
-        this.setState(GameState.RUNNING);
-
         for (int i = 0; i < this.rounds; i++) {
-            // TODO: 자동차 이동 구현
+            this.carRegistry.moveCars();
         }
-
-        this.setState(GameState.FINISHED);
-        List<Car> winners = this.getWinners();
-
         // TODO: 우승자 출력
     }
 
     public List<Car> getWinners() {
-        if (this.getState() != GameState.FINISHED) {
-            throw new IllegalStateException("Winners must be calculated within finish state.");
-        }
-
-        int bestPosition = -1;
-
-        for (Car car : cars) {
-            int position = car.getPosition();
+        int bestPosition = this.getBestPosition();
 
-            if (position > bestPosition) {
-                bestPosition = position;
-            }
-        }
-
-        List<Car> winners = new ArrayList<>();
-
-        for (Car car : cars) {
-            if (car.getPosition() == bestPosition) {
-                winners.add(car);
-            }
-        }
-
-        return winners;
+        return carRegistry.getRegisteredCars().stream().filter((car) -> car.getPosition() == bestPosition).toList();
```

Comment:
```suggestion
        return carRegistry.getRegisteredCars()
                .stream()
                .filter((car) -> car.getPosition() == bestPosition)
                .toList();
```

요거는 가독성을 위해 보통 `.`을 한줄에 하나씩만 사용하고, 줄바꿈을 하는 편이에요